### DISCUSSION TITLE
Can I import markdown with custom label/relation? Can I import day note from my diary?

### DISCUSSION THREAD

**Discussion by ZickShen** [TriliumNext/Trilium#3186] [Q&A]
Can I import markdown with custom label/relation if I generate a `!!!meta.json` with just some attributes like label?

I used to write my diary with basic markdown file, now I wrote a script to order them the same way as trilium. When I import them trilium won't add dateNote label, I wonder if I can write a script to generate a `!!!meta.json` so that imported note will have that label?

**Comment by ZickShen** [comment]
I checked exported `!!!meta.json` and found out that the only thing I can't guaranteed to work is noteId related attributes like `noteId` and `notePath`.

**Comment by zadam** [comment]
Hi, yes, it's possible - Trilium allows you to export a note tree in Markdown and will generate the metadata too. The import then works as well.

**Reply by ZickShen** [reply]
I want to import markdown from other application, so I wonder if there's a way to generate the metadata by myself. I tried last night and it failed. Here's my code <https://gist.github.com/CSUwangj/871e54f227499ea73782bcb660750e55>

**Reply by ZickShen** [reply]
with metadata generate by the code, import process will get stuck.
